### PR TITLE
[Relay][VM] Add more passes to VMCompiler

### PR DESCRIPTION
**Pull request by wweic**
This matches `relay/backend/build_module.cc`

cc @zhiics @jroesch @icemelon9 @yongwww

### CODE REVIEW

File: src/relay/backend/vm/compiler.cc [hunk 1]
```diff
@@ -844,18 +845,63 @@ void VMCompiler::Compile(const Module& mod_ref,
...
-  transform::Sequential seq({transform::SimplifyInference(),
-                             transform::InlinePrimitives(),
-                             // TODO(@wweic): FuseOps pass currently don't handle Let
-                             // For now, we put FuseOps before ToANormalForm to enable it
-                             transform::FuseOps(),
-                             transform::ToANormalForm(),
-                             transform::LambdaLift(),
-                             transform::InlinePrimitives()});
+Module VMCompiler::OptimizeModule(const Module& mod, const TargetsMap& targets) {
+  // TODO(@icemelon9): check number of targets and build config
```
Comment (icemelon9) [inline src/relay/backend/vm/compiler.cc]: Can we check the relay build config?

Comment (wweic) [reply src/relay/backend/vm/compiler.cc]: done

File: src/relay/backend/vm/compiler.cc [hunk 2]
```diff
@@ -844,18 +845,62 @@ void VMCompiler::Compile(const Module& mod_ref,
...
+  pass_seqs.push_back(transform::FoldConstant());
+
+  pass_seqs.push_back(transform::FuseOps());
+  pass_seqs.push_back(transform::ToANormalForm());
+  pass_seqs.push_back(transform::LambdaLift());
+  pass_seqs.push_back(transform::InlinePrimitives());
+
+  transform::Sequential seq(pass_seqs);
+  transform::PassContext pass_ctx = PassContext::Current();
   tvm::With<relay::transform::PassContext> ctx(pass_ctx);
```
Comment (icemelon9) [inline src/relay/backend/vm/compiler.cc]: Add a todo here for heterogeneous pass.

Comment (wweic) [reply src/relay/backend/vm/compiler.cc]: done

**Pull request merged**